### ISSUE TITLE
The nodejs/chakra bundle does not work with Ubuntu on Windows.

### ISSUE THREAD

**Issue by restarian** [nodejs/node-chakracore#332]
build: _node-v8.1.2-nightly201706274be96a0e53-linux-x64.tar.xz_

The above build does not work on Ubuntu under Windows. The exact output from it is:
_> ./node
Out of Memory
Aborted (core dumped)_

_> uname -a
Linux DESKTOP-BCNCPSO 4.4.0-43-Microsoft #1-Microsoft Wed Dec 31 14:42:53 PST 2014 x86_64 x86_64 x86_64 GNU/Linux_

 The same build does work with a native install of Ubuntu however. The below pictures show the working and non-working output of the two different platforms.

**Also, it is worth noting that node/v8 is working fine on Ubuntu in WIndows. Below is the output.**
``
surge@DESKTOP-BCNCPSO:/mnt/c/Users/Rober/commonjs_projects/tester$ node
> process.versions
{ http_parser: '2.7.0',
  node: '7.10.0',
  v8: '5.5.372.43',
  uv: '1.11.0',
  zlib: '1.2.11',
  ares: '1.10.1-DEV',
  modules: '51',
  openssl: '1.0.2k',
  icu: '58.2',
  unicode: '9.0',
  cldr: '30.0.3',
  tz: '2016j' }
> surge@DESKTOP-BCNCPSO:/mnt/c/Users/Rober/commonjs_projects/tester$ uname -a
Linux DESKTOP-BCNCPSO 4.4.0-43-Microsoft #1-Microsoft Wed Dec 31 14:42:53 PST 2014 x86_64 x86_64 x86_64 GNU/Linux
``
**Non-functioning build on Ubuntu in Windows**
![capture](https://user-images.githubusercontent.com/1820503/27996862-b8f9966e-64b9-11e7-8874-77dc47ffa88e.PNG)

**The same file working on a native Ubuntu install**
![chakra_working](https://user-images.githubusercontent.com/1820503/27996874-3330977a-64ba-11e7-9d4b-561baada455b.jpg)

**Comment by jianchun** [comment]
This is probably https://github.com/Microsoft/BashOnWindows/issues/597. What's your Windows version? (That issue fixed in Creators update.)

**Comment by restarian** [comment]
Well I do have the creators update version of windows at 1703. 

![version_info](https://user-images.githubusercontent.com/1820503/28032505-bf30504e-6578-11e7-92c1-050c7b96f57f.png)

**Comment by obastemur** [comment]
@jianchun I'm doing a related stuff on Windows-Bash. If you didn't spend time on it already, taking this one.

**Comment by kfarnung** [comment]
@restarian Can you run `lsb_release -a` in the Linux shell?  Was this machine upgraded to 1703 and do you remember if you setup Ubuntu on Linux before the upgrade?

EDIT:  Actually, I'm seeing this on my own machine with 1703, I'll let @obastemur investigate further.

**Comment by obastemur** [comment]
The problem should be fixed on Windows Bash end. Keeping this open and will update once the problem is fixed.

**Comment by jianchun** [comment]
@kfarnung My box works fine though (170513-2252):

```
xjc@xjc1:~/node-v8.1.2-nightly201706274be96a0e53-linux-x64$ bin/node
> process.versions
{ http_parser: '2.7.0',
  node: '8.1.2-nightly201706274be96a0e53',
  chakracore: '1.5.1.0',
  uv: '1.12.0',
  zlib: '1.2.11',
  ares: '1.10.1-DEV',
  modules: '57',
  openssl: '1.0.2l',
  icu: '59.1',
  unicode: '9.0',
  cldr: '31.0.1',
  tz: '2017b' }
```

**Comment by kfarnung** [comment]
On my SP4 device I'm seeing that same as @restarian.  One thing to note is that I had the Linux Subsystem enabled when I upgraded from 1607 to 1703 just the other day (standard Windows Update) and I uninstalled/reinstalled the Ubuntu image using:

```
> lxrun /uninstall /full
> bash
```

Maybe there's something about upgrades that's causing issues?  On my insider build machine I'm not seeing any issues.
